beacon-chain/rpc/service_test.go

```diff
@@ -42,30 +42,6 @@ func TestLifecycle_OK(t *testing.T) {
 
 }
 
-func TestRPC_BadEndpoint(t *testing.T) {
```

Comment:
This looks unrelated, but it was so flaky and annoying that I support removing it in this PR.

Reply:
It also wasn't really testing much - it wasn't really behavior we cared about testing in the first place